### ISSUE TITLE
Where to find input transforms info for pretrained models other than slowfast?

### ISSUE THREAD

**Issue by richardkxu** [facebookresearch/pytorchvideo#121]
Hi, I have a question regarding loading the pretrained models. In the [torch hub inference tutorial](https://github.com/facebookresearch/pytorchvideo/blob/9d0ca900f0427ed9b47b6182ad05f75c0e66274b/website/docs/tutorial_torchhub_inference.md#input-transform), it mentions:
```
NOTE: The input transforms are specific to the model. If you choose a different model than 
the example in this tutorial, please refer to the code provided in the Torch Hub documentation 
and copy over the relevant transforms:

####################
# SlowFast transform
####################

side_size = 256
mean = [0.45, 0.45, 0.45]
std = [0.225, 0.225, 0.225]
crop_size = 256
num_frames = 32
sampling_rate = 2
frames_per_second = 30
alpha = 4

# The duration of the input clip is also specific to the model.
clip_duration = (num_frames * sampling_rate)/frames_per_second

transform =  ApplyTransformToKey(
    key="video",
    transform=Compose(
        [
            UniformTemporalSubsample(num_frames),
            Lambda(lambda x: x/255.0),
            NormalizeVideo(mean, std),
            ShortSideScale(
                size=side_size
            ),
            CenterCropVideo(crop_size),
            PackPathway()
        ]
    ),
)
```

However, I was only able to find input transforms info for "Slow", "SlowFast", and "X3D". Where can I find the input transform info for other pretrained models provided in the [model zoo](https://github.com/facebookresearch/pytorchvideo/blob/main/docs/source/model_zoo.md) such as "CSN", "R(2+1)D", and "MViT"? 

I guess I can get the num_frames and sampling_rate from the "frame length x sample rate" listed in the table, but not the other info. I really appreciate if someone can point me to the right place to look up such info or it can be provided in the documentation/torch hub!

**Comment by dilky97** [comment]
Hi! Can you explain what is frame length x sample rate means? I'm struggling to understand what it does

**Comment by saniazahan** [comment]
@lyttonhao Hi I couldn't find the mean and std for normalization in the config file for MViT. Could you direct me where can I get it.
